app/src/features/projects/view/ProjectAttachments.tsx

```diff
@@ -123,8 +138,14 @@ const ProjectAttachments: React.FC<IProjectAttachmentsProps> = () => {
 
       <SecurityDialog
         open={securityDialogOpen}
-        onAccept={() => {
-          setSecurityDialogOpen(false);
+        onAccept={(securityReasons) => {
+          if (selectedAttachmentRows.length > 0) {
+            // formik form is retuning array of strings not numbers if printed out in console
+            // linter wrongly believes formik to be number[] so wrapped map in string to force values into number[]
```

Comment:
Ah yeah, because the value attribute of a checkbox either doesn't accept numbers or always converts them to strings (either way they only support strings, which is dumb).